roles/candlepin/defaults/main.yml

```diff
@@ -15,6 +15,7 @@ candlepin_ciphers:
 candlepin_container_image: quay.io/ehelms/candlepin
 candlepin_container_tag: 4.4.14
 
+candlepin_db_user: candlepin
```

Comment:
This looks like you were experimenting with something else and ended up committing it too.

Reply:
Correct - I started doing all of them and then decided to just do one to see what it looked and felt like.